docs/source/index.rst

```diff
@@ -10,336 +10,13 @@ AlphaPeel
-  id2    0.0399    0.0000    0.9545    0.0000
-  id2    0.0000    0.0000    0.0000    0.0000
-  id3    0.9998    0.0001    0.0001    1.0000
-  id3    0.0000    0.4999    0.4999    0.0000
-  id3    0.0000    0.4999    0.4999    0.0000
-  id3    0.0001    0.0001    0.0001    0.0000
-  id4    1.0000    1.0000    0.0000    1.0000
-  id4    0.0000    0.0000    0.0000    0.0000
-  id4    0.0000    0.0000    0.0000    0.0000
-  id4    0.0000    0.0000    1.0000    0.0000
-
-Dosage file
--------------
-
-The dosage file gives the expected allele dosage for the alternative (or minor) allele for each individual. The first value in each line is the individual ID. The remaining values are the allele dosages at each loci. These values will be between 0 and 2.
-
-Example: ::
-
-  1    0.0003    1.0000    1.0000    0.0001
-  2    1.0000    0.0000    1.0000    0.0000
-  3    0.0003    1.0000    1.0000    0.0001
-  4    0.0000    0.0000    2.0000    0.0000
-
-
-Segregation file
-----------------
-
-The segregation file gives the joint probability of each pattern of inheritance. There are four lines for each individual representing the probability of inheriting: 
-
-  1. the grand **paternal** allele from the father and the grand **paternal** allele from the mother
-  2. the grand **paternal** allele from the father and the grand **maternal** allele from the mother
-  3. the grand **maternal** allele from the father and the grand **paternal** allele from the mother
-  4. the grand **maternal** allele from the father and the grand **maternal** allele from the mother
-
-Example: ::
-
-  id1    1.0000    0.9288    0.9583    0.9834
-  id1    0.0000    0.0149    0.0000    0.0000
-  id1    0.0000    0.0554    0.0417    0.0166
-  id1    0.0000    0.0009    0.0000    0.0000
-  id2    0.9810    0.9842    1.0000    0.9971
-  id2    0.0174    0.0158    0.0000    0.0013
-  id2    0.0016    0.0000    0.0000    0.0016
-  id2    0.0000    0.0000    0.0000    0.0000
-  id3    0.0164    0.0149    0.0000    0.0065
-  id3    0.9259    0.9288    0.9582    0.9769
-  id3    0.0010    0.0009    0.0000    0.0001
-  id3    0.0567    0.0554    0.0417    0.0165
-  id4    0.0002    0.0000    0.0002    0.0004
-  id4    0.0015    0.0000    0.0019    0.0041
-  id4    0.1189    0.1179    0.1052    0.0834
-  id4    0.8794    0.8821    0.8927    0.9122
-
-Parameter files
----------------
-
-|ap| outputs three parameter files, ``.maf``, ``.seqError``, ``.genoError``. These give the minor allele frequency, sequencing error rates, and genotyping error rates used. All three files contain a single column with an entry for each marker. 
-
-Example ``.maf`` file for four loci: 
-::
-
-  0.468005
-  0.195520
-  0.733061
-  0.145847
-
+|Software| is program to call, impute, and phase genotypes using a pedigree in potentially very large livestock populations. A complete description of the methods is given in Whalen et al. (2018; http://dx.doi.org/10.1186/s12711-018-0438-2).
```

Comment:
@XingerTang I think we should move this last part (the short summary/intro of AlphaPeel) of `index.rst`, to the top -  current layout looks odd.

<img width="386" alt="Screenshot 2023-07-18 at 21 31 39" src="https://github.com/AlphaGenes/AlphaPeel/assets/725834/e22284ba-76fc-4ffd-ae41-363a70c2d28e">